### DISCUSSION TITLE
Icons and Colors documentation missing

### DISCUSSION THREAD

**Discussion by madrigal1** [gokcehan/lf#1156] [Q&A] [answered]
Its very hard to understand how to setup icons and colors. the wiki is completely missing all this info.
if someone could restore the old icons docs would be great

**Comment by gokcehan** [comment, marked as answer]
@madrigal1 I have now added a wiki page for [colors and icons](https://github.com/gokcehan/lf/wiki/Colors-and-Icons) configuration. Let me know there is something not clear.

**Reply by madrigal1** [reply]
Thanks so much for the fast reply and adding the docs :)
